inspirehep/bat/pages/holding_panel_literature_list.py

```diff
@@ -49,8 +49,28 @@ def click_first_record():
             (By.XPATH, '(//div[@class="ng-scope"])[2]')))
 
 
-def load_submission_record(input_data):
-    def _load_submission_record():
+def load_submitted_record():
+    def _force_load_record():
+        try:
+            # click the abstract link
+            WebDriverWait(Arsenic(), 10).until(
+                EC.visibility_of_element_located(
+                    (By.XPATH, '//div[@class="row hp-item ng-scope"][1]/div/div/div[2]/holding-pen-template-handler/div[3]/a')
+                )
+            ).click()
+            # take the record
+            record = WebDriverWait(Arsenic(), 10).until(
+                EC.visibility_of_element_located(
+                    (By.XPATH, '//div[@class="row hp-item ng-scope"][1]')
+                )
+            ).text
+        except (ElementNotVisibleException, WebDriverException):
+            go_to()
+            return _force_load_record()
+
+        return record
```

Comment:
I moved and duplicate the function `_force_load_record` because we force the reload in different ways in according to the status of an article. In addition the html code that show a `submitted` article and a `completed` article in the list is different  for this reason it requires a different xpath command